app/controllers/katello/api/v2/content_view_versions_controller.rb

```diff
@@ -105,20 +106,33 @@ def export_api_status
       render json: { api_usable: SmartProxy.pulp_primary.pulp3_repository_type_support?(Katello::Repository::YUM_TYPE) }, status: :ok
     end
 
-    api :POST, "/content_view_versions/:id/export", N_("Export a content view version")
+    api :POST, "/content_view_versions/:id/export_legacy", N_("Export a content view version")
     param :id, :number, :desc => N_("Content view version identifier"), :required => true
-    param :destination_server, String, :desc => N_("Destination Server name, required for Pulp3")
+    param :destination_server, String, :desc => N_("Destination Server name, required for Pulp3"), :required => true
     param :chunk_size_mb, :number, :desc => N_("Chunk export-tarfile into pieces of chunk_size mega bytes. Relevant only for Pulp 3 repositories"), :required => false
```

Comment:
```suggestion
```

Reply:
@parthaa GitHub is breaking this suggestion, when I try to edit the comment all I see is `suggestion`. 

What needs to be updated here?

Reply:
Fixed